README.md

```diff
@@ -29,16 +29,7 @@ You can skip the hook by adding `--no-verify` to your `git push`.
 - Download and install [Ruby-Build](https://github.com/rbenv/ruby-build#installing-as-an-rbenv-plugin-recommended).
 - Install the appropriate Ruby version by running `rbenv install [version]` where `version` is the one located in [.ruby-version](.ruby-version)
 
-### 2- Installing Yarn
-
-```bash
-  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-  echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-  sudo apt remove cmdtest # Some Ubuntu systems comes with cmdtest installed by default.
-  sudo apt-get update && sudo apt-get install yarn
-```
-
-### 3- Installing Rails gems
+### 2- Installing Rails gems
```

Comment:
Can we delete this incorrect badge?
```
[![Codestats](http://codestats-url/organizations/your-organization/projects/rails-bootstrap/badge)](http://codestats-url/organizations/your-organization/projects/rails-bootstrap/badge)
```

Reply:
I'm not sure what we should do with Codestats in general though